tests/ui/get_last_with_len.stderr

```diff
@@ -0,0 +1,10 @@
+error: unknown clippy lint: clippy::get_last_with_len
+  --> $DIR/get_last_with_len.rs:1:9
+   |
+LL | #![warn(clippy::get_last_with_len)]
```

Comment:
You have to `register_late_pass(box UseLast)` in `lib.rs`. See https://github.com/rust-lang/rust-clippy/blob/master/doc/adding_lints.md#lint-declaration

Reply:
Thanks. I always forget about that, since it's the only thing that's not automated.